pkg/api/node.go

```diff
@@ -121,6 +125,64 @@ func (m *nodeMetrics) Get(ctx context.Context, name string, opts *metav1.GetOpti
 	return &nodeMetrics[0], nil
 }
 
+func (m *nodeMetrics) ConvertToTable(ctx context.Context, object runtime.Object, tableOptions runtime.Object) (*metav1beta1.Table, error) {
```

Comment:
Is this function part of some interface? can we add this interface to the list on line 46?

Reply:
Yes, I forgot that, thank you.